resources/leiningen/new/jcf/resources/config.edn

```diff
@@ -0,0 +1,12 @@
+{:http
+ {:connection-timeout 2000
+  :pool-timeout 1000
+  :request-timeout 1000}
+
+ :logger
+ {:level #profile {:default :debug :uberjar :debug}
+  :path #profile {:default "logs/dev.log"
+                  :test "logs/test.log"
+                  :uberjar nil}}
```

Comment:
This `:logger `config is for Timbre, is it not? And we're not using Timbre or at least we probably shouldn't be.